src/main/kotlin/su/arlet/business1/core/Article.kt

```diff
@@ -1,13 +1,29 @@
 package su.arlet.business1.core
 
 import jakarta.persistence.*
+import jakarta.validation.constraints.NotEmpty
+import su.arlet.business1.core.enums.ArticleStatus
 
 @Entity
 @Table(name = "articles")
 class Article(
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    val id: Long,
-    val title: String,
-    val text: String,
+    var id: Long = 0,
+
```

Comment:
Also need field like clarificationText in AdRequest (when editor says what journalist need to fix)